src/test/java/org/traccar/protocol/MegastekProtocolDecoderTest.java

```diff
@@ -39,6 +39,9 @@ public void testDecode() throws Exception {
         verifyPosition(decoder, text(
                 "$MGV002,869152024446923,869152024446923,S,240816,151631,A,5053.83335,N,00424.05702,E,00,10,00,0.88,2.645,76.09,22.7,,206,01,07D1,6600,28,,,,,,,,,01,100,Timer;!"));
 
+        verifyPosition(decoder, text(
+                "0143$MGV002,869152024261564,,R,220621,120804,V,5152.09429,N,01051.32158,E,00,00,00,99.9,,,,,232,10,A5AE,5A4908,22,0000,0000,0,,,,,,00,093,Timer,,;!"));
```

Comment:
Is this a real sample you got from your device?

Reply:
This is a real sample I got from my device yes. I anonymized certain details like coordinates though. I also tested it on my instance and it worked flawlessly.